cmd/chronoctl/add.go

```diff
@@ -2,17 +2,18 @@ package main
 
 import (
 	"context"
+	"strings"
 
 	"github.com/influxdata/chronograf"
 )
 
 type AddCommand struct {
-	BoltPath string  `short:"b" long:"bolt-path" description:"Full path to boltDB file (e.g. './chronograf-v1.db')" env:"BOLT_PATH" default:"chronograf-v1.db"`
-	ID       *uint64 `short:"i" long:"id" description:"Users ID. Must be id for existing user"`
-	Username string  `short:"n" long:"name" description:"Users name. Must be Oauth-able email address or username"`
-	Provider string  `short:"p" long:"provider" description:"Name of the Auth provider (e.g. google, github, auth0, or generic)"`
-	Scheme   string  `short:"s" long:"scheme" description:"Authentication scheme that matches auth provider (e.g. oauth or ldap)"`
-	//Organizations string `short:"o" long:"orgs" description:"A comma separated list of organizations that the user should be added to"`
+	BoltPath      string  `short:"b" long:"bolt-path" description:"Full path to boltDB file (e.g. './chronograf-v1.db')" env:"BOLT_PATH" default:"chronograf-v1.db"`
+	ID            *uint64 `short:"i" long:"id" description:"Users ID. Must be id for existing user"`
+	Username      string  `short:"n" long:"name" description:"Users name. Must be Oauth-able email address or username"`
+	Provider      string  `short:"p" long:"provider" description:"Name of the Auth provider (e.g. google, github, auth0, or generic)"`
+	Scheme        string  `short:"s" long:"scheme" description:"Authentication scheme that matches auth provider (e.g. oauth or ldap)"`
+	Organizations string  `short:"o" long:"orgs" description:"A comma separated list of organizations that the user should be added to"`
```

Comment:
By name or id

Reply:
ID, but I can do name if that's easier.

Reply:
Does list show the name or the id? The user will just need access to the info in case they are totally locked out.

Reply:
list shows the ids

Reply:
1. the work flow I was imagining was look to see all users
2. grab an existing users list of ids
3. modify them
4. add new super admin with the modified list